src/qtcore/qmlstructure.js

```diff
@@ -255,6 +255,9 @@ function loadParser() {
           throw new Error('Could not load QmlWeb parser!');
       }
       (new Function(xhr.responseText))();
+      // TODO: remove in 0.2, qmlweb.parser should set to QmlWeb if QmlWeb is defined
+      QmlWeb.qmlweb_parse = QmlWeb.qmlweb_parse || qmlweb_parse;
+      QmlWeb.qmlweb_jsparse = QmlWeb.qmlweb_jsparse || qmlweb_jsparse;
```

Comment:
What do you think about renaming that to `QmlWeb.parse` and `QmlWeb.parseJs`? :)